### DISCUSSION TITLE
Welcome to FVT-Baylor-University Discussions!

### DISCUSSION THREAD

**Discussion by GarnesDoug** [GarnesDoug/FVT-Baylor-University#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lee-tamucc** [comment]
Hello, I was wondering about the merge queries. It seems that these mimic the student submission. Is there some other code you can share for the program submission?
